src/node.cc

```diff
@@ -94,6 +94,10 @@
 #include <unicode/uvernum.h>
 #endif
 
+#if defined(NODE_REPORT)
```

Comment:
I am able to see both the styles in the code base! Unless we have a best practice in the guidelines, I desire to keep it as is?

Reply:
I thought about making the exact same comment as @lundibundi :smile: I didn’t because it doesn’t matter much to me, in the end, but yes, `#ifdef` is the much more common style (537 vs 33 occurrences).

Reply:
I'm fine with both, though #ifdef look a bit better to me :smile:.

Feel free to push resolve with whichever solution you prefer.

Reply:
followed the popular way!